modules/news/news.service.js

```diff
@@ -5,6 +5,7 @@ const {
   PHOTO_NOT_FOUND,
 } = require('../../error-messages/news.messages');
 const { uploadLargeImage } = require('../upload/upload.utils');
+const { deleteFile, deleteFiles } = require('../upload/upload.service');
```

Comment:
this import should be like this  const uploadService = require('../upload/upload.service');